cScripts/Loadouts/CfgLoadouts_Common.hpp

```diff
@@ -15,7 +15,7 @@ class CommonBlufor {
     //preLoadout = "";
     //postLoadout = "";
 
-    //role = "";
+    //equipmentTags[] = {""role_;XXxENDLINEXXx
```

Comment:
> role_;XXxENDLINEXXx 

Unsure what this is

Reply:
Haha ups! my find replace did not get translated propperly xD